File: sentimental-mario-less/mario.py
import cs50

# Get the pyramid height from the user
height = cs50.get_int("Height: ")
while height <= 0 or height > 8:
    height = cs50.get_int("Height: ")

for i in range(height):

    # Loop for printing spaces
    for j in range(i + 1, height):
        print(" ", end = '')

    # Loop for printing bricks of pyramid
    for k in range(i + 1, 0, -1):
        print("#", end = '')
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int get_height(void);

int main(void)
{
    // get the pyramid's height from user
    int height = get_height();

    for (int i = 0; i < height; i++)
    {
        // loop for printing spaces
        for (int j = i + 1; j < height; j++)
        {
            printf(" ");
        }
        // loop for printing bricks of pyramid
        for (int k = i + 1; k > 0; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_height(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    return height;
}
